isthmus/src/test/java/io/substrait/isthmus/DuplicateFunctionUrnTest.java

```diff
@@ -0,0 +1,187 @@
+package io.substrait.isthmus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import io.substrait.expression.Expression;
+import io.substrait.extension.SimpleExtension;
+import io.substrait.isthmus.expression.AggregateFunctionConverter;
+import io.substrait.isthmus.expression.ScalarFunctionConverter;
+import io.substrait.isthmus.expression.WindowFunctionConverter;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.List;
+import java.util.Optional;
+import org.apache.calcite.rex.RexBuilder;
+import org.apache.calcite.rex.RexCall;
+import org.apache.calcite.rex.RexNode;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.apache.calcite.sql.fun.SqlTrimFunction.Flag;
+import org.junit.jupiter.api.Test;
+
+/** Tests to reproduce #562 */
+public class DuplicateFunctionUrnTest extends PlanTestBase {
```

Comment:
I think for JUnit 5 the test class can be package private:

```suggestion
class DuplicateFunctionUrnTest extends PlanTestBase {
```